test/wallet.test.ts

```diff
@@ -0,0 +1,339 @@
+import { expect, describe, it } from '@jest/globals';
```

Comment:
The docs seem to indicate that importing the Jest globals is necessary if you plan on using ESM for Node, but since 1) we don't support current versions of Node yet and 2) we haven't configured TypeScript to spit out ESM, it doesn't seem like we need this yet. Was there a specific reason why you wanted to use this style?